ScienceJournal/Extensions/TimeInterval+ScienceJournal.swift

```diff
@@ -35,10 +35,19 @@ extension TimeInterval {
   }
 
   /// Returns the time interval as an accessible string in component format with hours, minutes and
-  /// seconds without abbreviation.
-  /// Example: "One hour 46 minutes 11 seconds"
+  /// seconds without abbreviation.  When there are fractional seconds present on the time interval,
+  /// this will include them as milliseconds.
+  /// Example: "One hour 46 minutes 11 seconds" and "One second 250 milliseconds".
   var accessibleDurationString: String {
-    return TimeInterval.accessibleIntervalFormatter.string(from: self) ?? ""
+    return TimeInterval.accessibleIntervalFormatter.string(from: self, appending: fractional)
```

Comment:
We're currently discussing this path because of a few issues:

1. In an ideal world, we'd use our existing DateComponentsFormatter to simply use milliseconds since it does the localization for us including locale ordering, etc. But obviously DCF doesn't support anything smaller than seconds.
2. The issue with appending this formatted string onto the end of the existing a11y string is that, in some locales, this might not be correct. We're not entirely sure, but given the history of how locales work it's entirely possible there's a locale in which milliseconds come first, perhaps.
3. We don't have a good solution to present instead 😅

We'll discuss a bit more internally and circle back to you on this. Greatly appreciate your PR and your patience. Thanks!